togetherly/lib/views/screens/child_home.dart

```diff
@@ -11,27 +16,63 @@ class ChildHomePage extends StatefulWidget {
 }
 
 class _ChildHomePageState extends State<ChildHomePage> {
+  final Child currentChild = const Child(
+    id: 1,
+    familyId: 1,
+    name: 'Emma',
+    icon: ProfileIcon.bear,
+    totalPoints: 45,
+  );
+
+  Future<void> buildDialog(BuildContext context) {
+    return showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return const NewChoreDialog();
+      },
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
-    return const Padding(
-      padding: AppWidgetStyles.appPadding,
-      child: SingleChildScrollView(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.start,
-          children: <Widget>[
-            Points(),
-            ChoreList(
-              title: 'Today',
-            ),
-            SizedBox(height: 20.0),
-            ChoreList(
-              title: 'Upcoming',
-            ),
-            SizedBox(height: 20.0),
-            ChoreList(
-              title: 'Overdue',
-            ),
-          ],
+    return Consumer<ChoreProvider>(
```

Comment:
Does the consumer need to cover the entire page? Or could it be moved down the hierarchy to wrap just the chore list widget? Or taking that further, could it be moved inside the chore list widget? The lower in the hierarchy the better, since every state change will trigger a rebuild of the tree beneath the consumer widget.